ext/session/session.c

```diff
@@ -1353,15 +1402,17 @@ static void ppid2sid(zval *ppid) {
 	}
 }
 
-PHPAPI void php_session_reset_id(void) /* {{{ */
+
+/* Made to return int from 7.1, previously void */
```

Comment:
We don't need to comment on this in the code, this should be in the manual and UPGRADING.